src/h3lib/lib/vertex.c

```diff
@@ -68,13 +69,17 @@ static H3Error vertexRotations(H3Index cell, int *out) {
     if (_isBaseCellPentagon(baseCell)) {
         // Find the appropriate direction-to-face mapping
         PentagonDirectionFaces dirFaces;
-        // Excluding from branch coverage as we never hit the end condition
-        for (int p = 0; p < NUM_PENTAGONS; p++) {  // LCOV_EXCL_BR_LINE
+        // We never hit the end condition
+        int p = 0;
+        for (; p < NUM_PENTAGONS; p++) {
             if (pentagonDirectionFaces[p].baseCell == baseCell) {
                 dirFaces = pentagonDirectionFaces[p];
                 break;
             }
         }
+        if (p == NUM_PENTAGONS) {
```

Comment:
Should this be `NEVER(p == NUM_PENTAGONS)`? This case is supposedly unreachable, right?

Reply:
I'm not sure that would satisfy the compile error about dirFaces being uninitialized. We could check.